### PR TITLE
replaced with logical_date 

### LINKED ISSUE #935: Replace `execution_date` with `logical_date`
## Description

We are receiving deprecation warnings like the following in production:

```
[2022-12-29, 00:02:17 UTC] {logging_mixin.py:120} WARNING - /home/airflow/.local/lib/python3.10/site-packages/airflow/utils/context.py:204 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
```

It seems like these are mostly coming from the `common.slack` module, but we should replace instances of `execution_date` with `logical_date` to account for this.

### PR DESCRIPTION
**Pull request by sora-san45**
## Fixes

Fixes #935 by @sora-san45

## Description

Replaced execution_date with logical_date in common.slack to prevent deprecation in future 
Added attribute delay = 0.1 for class WikimediaCommonsDataIngester 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.
- [x] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ class WikimediaCommonsDataIngester(ProviderDataIngester):
         "audio": prov.WIKIMEDIA_AUDIO_PROVIDER,
     }
     endpoint = f"https://{HOST}/w/api.php"
+    delay =0.1
```
Comment (AetherUnbound) [inline openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py:49]: This change was suggested only for testing in https://github.com/WordPress/openverse-catalog/issues/727 and should not be included in this changeset:
```suggestion
```

**Review by AetherUnbound — CHANGES_REQUESTED** [review]
Thank you for the contribution @sora-san45! It looks like there are a number of files that either changed modes or were deleted. Those should not be part of the suggested changeset to address #935. Please revert those changes and the one suggested below.

File: openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py (hunk 1 above)
Comment (sora-san45) [reply openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py]: Hey I have removed the delay=0.1 in the latest commit

Comment (AetherUnbound) [reply openverse_catalog/dags/providers/provider_api_scripts/wikimedia_commons.py]: I still see 17 files total modified by the diff, when I believe only `slack.py` should be affected by this change. Can you revert the changes that have been made to the other files? Here is a guide for [undoing things in git](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things) if you haven't encountered reverting changes before. Let us know if you need any help doing that 🙂

**Review by AetherUnbound — CHANGES_REQUESTED** [review]
This looks like it has a number of unrelated files affected, see my comment [here](https://github.com/WordPress/openverse-catalog/pull/950#discussion_r1067196404)

**Pull request closed without merging**